osd_support/osd-managed-resources.adoc

```diff
@@ -28,7 +28,7 @@ include::https://raw.githubusercontent.com/openshift/managed-cluster-config/mast
 [id="osd-add-on-managed-namespaces"]
 == {product-title} add-on namespaces
 
-{product-title} add-ons are services available for installation after cluster installation. These additional services include AWS CloudWatch, Red Hat CodeReady Workspaces, Red Hat OpenShift API Management, and Cluster Logging Operator. Any changes to resources within the following namespaces might be overridden by the add-on during upgrades, which can lead to unsupported configurations for the add-on functionality.
+{product-title} add-ons are services available for installation after cluster installation. These additional services include Red Hat CodeReady Workspaces, Red Hat OpenShift API Management, and Cluster Logging Operator. Any changes to resources within the following namespaces might be overridden by the add-on during upgrades, which can lead to unsupported configurations for the add-on functionality.
```

Comment:
FYI, CodeReady Workspaces is called "Red Hat OpenShift Dev Spaces" now. You can use `{openshift-dev-spaces-productname}` if you also include `_attributes/common-attributes.adoc`.